Makefile

```diff
@@ -71,6 +71,11 @@ go-vet:
 check-dockerfiles:
 	./hack/check-dockerfiles.sh
 
+check-images-availability: bin/crane
```

Comment:
i think adding a comment to `gen-manifests` prompting users to run this target is a good idea.